openpype/lib/transcoding.py

```diff
@@ -1037,3 +1037,46 @@ def convert_ffprobe_fps_to_float(value):
     if divisor == 0.0:
         return 0.0
     return dividend / divisor
+
+
+def convert_colorspace(
+    input_path,
+    out_filepath,
+    config_path,
+    source_colorspace,
+    target_colorspace,
+    logger=None
+):
+    """Convert source files from one color space to another.
+
+    Filenames of input files are kept so make sure that output directory
+    is not the same directory as input files have.
+    - This way it can handle gaps and can keep input filenames without handling
+        frame template
+
+    Args:
+        input_path (str): Paths that should be converted. It is expected that
+            contains single file or image sequence of samy type.
```

Comment:
Docstring is confusing? Is it a single path or multiple? Paths? or path?

Also, why is it `input_path` versus `out_filepath`

I'd expect either:
```
input_path
output_path
```
or
```
in_filepath
out_filepath
```

Reply:
Cleaned up.